index.html

```diff
@@ -20,26 +21,161 @@
         </nav>     
     </header>
     <main >
-        <section class="text-container container">
-            <h1 class="welcome"> Hey There. I'm Fernando</h1>            
-            <p class="profession">I'm Software Developer</p>
-            <p class="xp">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</p>
+        <section class="presentation">
+            <section class="text-container container">
+                <h1 class="welcome"> Hey There. I'm Fernando</h1>            
+                <p class="profession">I'm Software Developer</p>
+                <p class="xp">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </section>        
+            <ul class="social-media">
+                <li>
+                    <a href="#"><img class="github" src="img/github.png" alt="github"></a>
+                </li>
+                <li>
+                    <a href="#"><img class="vector" src="img/vector.png" alt="vector"></a>
+                </li>
+                <li>
+                    <a href="#"><img class="twitter" src="img/twitter.png" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="#"><img class="msocial" src="img/msocial.png" alt="github"></a>
+                </li>
+            </ul>
+        </section>
+        
+
+        <section class="works-section">
+            <h2 class="title-works">My Recent Works</h2>            
+            <div class="vector"></div>
+        </section>
+
+        <section class="stories-section">
+            <img src="img/imgplaceholder.png" alt="placeholderImg">
+            <h2>Multi-Post Stories</h2>
+            <p class="story">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="techs-container">
+                <p class="techs">css</p>
+                <p class="techs">html</p>
+                <p class="techs">bootstrap</p>
+                <p class="techs">Ruby</p>
+            </div>
+            <input class="button" type="submit" value="See Project">
+        </section>
+        <section class="projects">
+            <section class="projects-container">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="techs-container">                
```

Comment:
**The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.**
- Kindly change your `<p class="techs">` instead `<li class="techs">`.